pdr_backend/lake/etl.py

```diff
@@ -218,11 +234,17 @@ def _calc_bronze_start_end_ts(self):
             else datetime.strptime(self.ppss.lake_ss.st_timestr, "%Y-%m-%d_%H:%M")
         )
 
+        fin_timestamp = (
```

Comment:
this is very confusing. It seems in one case, you assign this variable the result of strptime, and in another case the value of strftime. But those are not the same data type. Should this be a string or a date time?

Reply:
I guess strptime, some of the canges are from other merges in the base PR